install/0000_00_cluster-version-operator_01_admingate_configmap.yaml

```diff
@@ -1,5 +1,9 @@
 apiVersion: v1
 kind: ConfigMap
+data:
+  ack-4.8-kube-122-api-removals-in-4.9: |
```

Comment:
nit: doesn't really matter, but with `4.8` and `4.9` both including an explicit `.`, it seems strange to use `122` for Kube instead of `1.22`.  Maybe change to `ack-4.8-kube-1.22-api-removals-in-4.9`?

Reply:
Agree. No reason that won't work and although that's what we specified in the enhancement that's shouldn't matter.

Reply:
changed